content/uiux/concepts/design-thinking/design-thinking.md

```diff
@@ -0,0 +1,32 @@
+---
+Title: 'Design Thinking'
+Description: 'Design thinking is a design methodology that puts people at the center of every process and encourages designers to set aside assumptions.'
+Subjects:
+  - 'Web Design'
+Tags:
+  - 'UI'
+  - 'UX'
+CatalogContent:
+  - 'intro-to-ui-ux'
+  - 'paths/front-end-engineer-career-path'
+---
+
+**Design thinking** is a [design methodology](https://www.codecademy.com/resources/docs/uiux/design-methodologies) that puts people at the center of every process and encourages designers to set aside assumptions. This formal methodology has [developed across multiple disciplines since the 1960s](https://www.interaction-design.org/literature/article/design-thinking-get-a-quick-overview-of-the-history) and is commonly associated with the design and consulting firm [IDEO](https://designthinking.ideo.com/) and the Stanford School of Design (the [d.school](https://dschool.stanford.edu/)).
+
+![A diagram of the five phases of the design thinking process: Empathize, Define, Ideate, Prototype, and Test.](https://static-assets.codecademy.com/Courses/intro-to-ui-and-ux/key-methodologies/design-thinking-5-steps.png)
+
+Design thinking's core activities are ideation, inspiration, and implementation, which occur across the five stages of the process:
+
+1. **Empathize**: Understand the user and the landscape.
+2. **Define**: Define the problem and align with business goals and user needs.
+3. **Ideate**: Generate a range of ideas for possible solutions, emphasizing creativity.
+4. **Prototype**: Explore potential solutions by creating prototypes of the product to gather feedback.
+5. **Test**: Test the best solutions developed during prototyping. Prototyping or testing may lead to redefining the problem altogether. As with the other processes we've covered, this is an iterative cycle.
+
+Like the [double diamond model](https://www.codecademy.com/resources/docs/uiux/double-diamond-diagram), design thinking offers opportunities to focus on both [divergent thinking](https://www.codecademy.com/resources/docs/uiux/divergent-thinking) and [convergent thinking](https://www.codecademy.com/resources/docs/uiux/convergent-thinking) across its steps to encourage both creativity and problem solving.
+
+Design thinking isn't limited to creating new products&mdash;it can affect change at a systemic level. The term "[wicked problems](https://www.codecademy.com/resources/docs/uiux/wicked-problems)" was coined by design theorist Horst Rittel to describe the types of extremely complex, multi-dimensional problems that designers are often tasked to solve.
```

Comment:
```suggestion
Design thinking isn't limited to creating new products &mdash; it can affect change at a systemic level. The term "[wicked problems](https://www.codecademy.com/resources/docs/uiux/wicked-problems)" was coined by design theorist Horst Rittel to describe the types of extremely complex, multi-dimensional problems that designers are often tasked to solve.
```